api/src/test/java/org/openmrs/PersonTest.java

```diff
@@ -880,4 +880,22 @@ public PersonAddress build() {
 			return personAddress;
 		}
 	}
+
+	/**
+	 * @see Person#setDeathDate(Date)
+	 * @verifies set dead flag depending on deathDate presence
+	 */
+	@Test
+	public void setDeathDate_shouldSetDeadFlagDependingOnDeathDatePresence() {
+		Calendar birthdate = Calendar.getInstance();
+		birthdate.set(1990, Calendar.JUNE, 2);
+		Calendar deathDate = Calendar.getInstance();
+		deathDate.set(2000, Calendar.JUNE, 3);
+		Person person = new Person();
+		person.setBirthdate(birthdate.getTime());
```

Comment:
What could be the reason behind setting the birthdate? Is it needed for this test? Did you consider just using the test as it is in the ticket description? 😊